src/rexsdesktop/model/UserModel.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package rexsdesktop.model;
+
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.sql.Blob;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.Connection;
+import javax.imageio.ImageIO;
+import rexsdesktop.controller.User_;
+
+/**
+ *
+ * @author Eduardo
+ */
+public class UserModel{

```

Comment:
all of the methods of this class shouldn't be static

Reply:
also make an interface for this class and name it as `UserRepository`

Reply:
Okay, I'll change it.